app/services/square_manager/webhook_validator.rb

```diff
@@ -34,9 +34,17 @@ def call
         )
       )
 
+      apex_signature = Base64.strict_encode64(
+        OpenSSL::HMAC.digest(
+          'sha1',
+          ENV['APEX_WEBHOOK_SIGNATURE_KEY'],
+          string_to_sign
+        )
+      )
+
       # Hash the signatures a second time (to protect against timing attacks)
       # and compare them
-      unless [Digest::SHA1.base64digest(string_signature), Digest::SHA1.base64digest(think_chinatown_signature)].include?(Digest::SHA1.base64digest(callback_signature))
+      unless [Digest::SHA1.base64digest(string_signature), Digest::SHA1.base64digest(think_chinatown_signature), Digest::SHA1.base64digest(apex_signature)].include?(Digest::SHA1.base64digest(callback_signature))
```

Comment:
same might also be worth abstracting out `Digest::SHA1.base64digest(blah)`

Reply:
Ditto